src/fetchers/streak-fetcher.js

```diff
@@ -0,0 +1,123 @@
+import retryer from "../common/retryer.js";
+import { MissingParamError, request } from "../common/utils.js";
+
+const QUERY = `
+query streakInfo($userName:String!) { 
+  user(login: $userName){
+    name
+    contributionsCollection {
+      contributionCalendar {
+        totalContributions
+        weeks {
+          contributionDays {
+            contributionCount
+            date
+          }
+        }
+      }
+    }
+  }
+}
+`;
+
+const fetcher = async (variables, token) => {
```

Comment:
It would be great if this function had docstrings and typings. See

https://github.com/anuraghazra/github-readme-stats/blob/28b65928d2073c81b1d206e1197087e91d25778e/src/fetchers/stats-fetcher.js#L80-L90